usr/share/rear/lib/layout-functions.sh

```diff
@@ -530,10 +530,7 @@ get_disk_size() {
 
     local block_size=$(get_block_size ${disk_name%/*})
 
-    [ -r /sys/block/$disk_name/size ]
-    BugIfError "Could not determine size of disk $disk_name, please file a bug."
-
-    local nr_blocks=$( < /sys/block/$disk_name/size)
+    local nr_blocks=$(retry_command "cat /sys/block/$disk_name/size")
```

Comment:
I'm not sure that I understand this correctly. 
During my tests messages have been correctly logged into logfile so I really can't agree with "In case of error this change also drops the detailed error".